config/initializers/folio_client.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+if Settings.folio.enabled
```

Comment:
Do we need the enabled flag?

Reply:
Oh, for test environment. Can you add a comment either here or in `settings.yml` to make that explicit?

Reply:
Even better, I think? I removed the indirection. Have another look-see?